data/org.freedesktop.portal.Background.xml

```diff
@@ -32,70 +32,40 @@
 
         Supported keys in the @options vardict include:
-        <variablelist>
-          <varlistentry>
-            <term>handle_token s</term>
-            <listitem><para>
-              A string that will be used as the last element of the
-              @handle. Must be a valid object path element. See the
-              #org.freedesktop.portal.Request documentation for more
-              information about the @handle.
-            </para></listitem>
-          </varlistentry>
-          <varlistentry>
-            <term>reason s</term>
-            <listitem><para>
-              User-visible reason for the request.
-            </para></listitem>
-          </varlistentry>
-          <varlistentry>
-            <term>autostart b</term>
-            <listitem><para>
-              TRUE if the app also wants to be started automatically
-              at login.
-            </para></listitem>
-          </varlistentry>
-          <varlistentry>
-            <term>commandline as</term>
-            <listitem><para>
-              Commandline to use add when autostarting at login.
-              If this is not specified, the Exec line from the
-              desktop file will be used.
-            </para></listitem>
-          </varlistentry>
-          <varlistentry>
-            <term>dbus-activatable b</term>
-            <listitem><para>
-              If TRUE, use D-Bus activation for autostart.
-            </para></listitem>
-          </varlistentry>
-        </variablelist>
-
-       The following results get returned via the
-       #org.freedesktop.portal.Request::Response signal:
-
-      <variablelist>
-        <varlistentry>
-          <term>background b</term>
-          <listitem><para>
-            TRUE if the application is allowed to run in the background.
-          </para></listitem>
-        </varlistentry>
-        <varlistentry>
-          <term>autostart b</term>
-          <listitem><para>
-            TRUE if the application is will be autostarted.
-          </para></listitem>
-        </varlistentry>
-      </variablelist>
+
+        ``handle_token`` (``s``)
+          A string that will be used as the last element of the @handle.
+          Must be a valid object path element. See the
+          #org.freedesktop.portal.Request documentation for more
+          information about the @handle.
+
+        ``reason`` (``s``)
+          User-visible reason for the request.
+
+        ``autostart`` (``b``)
+          true if the app also wants to be started automatically at login.
+
+        ``commandline`` (``as``)
+          Commandline to use add when autostarting at login. If this is not
+          specified, the ``Exec`` key from the desktop file will be used.
+
+        ``dbus-activatable`` (``b``)
+          true to use D-Bus activation for autostart.
+
+        The following results get returned via the #org.freedesktop.portal.Request::Response signal:
```

Comment:
you probably already noticed but this notation no longer seems to generate the right links.

Reply:
Yes, this is likely an issue in gdbus-codegen; I've started looking at the generator.